app/src/vanilla/java/cm/aptoide/pt/app/view/EditorialFragment.java

```diff
@@ -504,4 +658,19 @@ private void handleStatusBar(boolean collapseState) {
       }
     }
   }
+
+  private boolean isScrollDown(int oldY, int newY) {
+    return newY > oldY;
+  }
+
+  private boolean isItemShown() {
+    return adapter.isItemShown(screenHeight, screenWidth);
```

Comment:
I don't really agree with this. Saving the viewholder on the adapter and getting the visibility through that doesn't seem a good. Is there an alternative?